### PR TITLE
Fifth section

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,7 +44,12 @@
     </div>
     <div class="site-wrapper">
       <section class="fifth-setion">
-        <div class="fifth-section-item">2</div>
+        <div class="fifth-section-item">
+          <h2 class="fifth-section-title">gdyugaldygulyfvayvf</h2>
+          <p class="fifth-section-text">
+            gsdyugfyulgfyusvfuyfve efugeufy fi g fu efuefgaelvfjs
+          </p>
```
Comment (blacksrc) [inline index.html]: Not a must or even should. But, it's better to use [lorem ipsum generator](https://www.lipsum.com/) tools to generate dummy text. Please keep that in mind for the future.
[Lorem ipsum wikipedia](https://en.wikipedia.org/wiki/Lorem_ipsum)

File: index.html [hunk 2]
```diff
@@ -44,7 +44,12 @@
     </div>
     <div class="site-wrapper">
       <section class="fifth-setion">
-        <div class="fifth-section-item">2</div>
+        <div class="fifth-section-item">
```
Comment (takrami) [inline index.html]: Do we need this div? Cant be removed?

File: index.html [hunk 3]
```diff
@@ -44,7 +44,12 @@
     </div>
     <div class="site-wrapper">
       <section class="fifth-setion">
```
Comment (takrami) [inline index.html:58]: Shouldn't have the dark background in full width? I mean the wrapper shouldn't be inside the section?

File: index.html (hunk 1 above)
Comment (hedita) [reply index.html]: Ok.

File: index.html (hunk 2 above)
Comment (hedita) [reply index.html]: Removed.

File: index.html (hunk 3 above)
Comment (hedita) [reply index.html:58]: I added it in section.

**Review by takrami — APPROVED** [review]

**Pull request merged**